packages/next/next-server/lib/utils.ts

```diff
@@ -191,6 +191,12 @@ export interface NextApiRequest extends IncomingMessage {
   body: any
 
   env: Env
+
+  preview: boolean
```

Comment:
This type doesn't seem correct. It can be undefined, right? It should match the getStaticProps behavior.